lang-dlang/ldc/01-liblphobos/prepare.stage2

```diff
@@ -0,0 +1,7 @@
+GDMD_WRAPPER="https://cdn.jsdelivr.net/gh/D-Programming-GDC/gdmd@ff2c97a47408fb71c18a2d453294d18808a97cc5/dmd-script"
+abinfo "Downloading gdmd wrapper ..."
```

Comment:
Please let ACBS handle this download.

Reply:
Since #6071 is merged, I will use that instead.

Reply:
Done.